public/docs/ts/latest/guide/architecture.jade

```diff
@@ -538,51 +522,49 @@ figure
 
   * A *provider* is a recipe for creating a service.
 
-  * We register *providers* with injectors.
+  * Register *providers* with injectors.
 
 .l-hr 
 
 .l-main-section
 :marked
   ## Wrap up
 
-  We've learned just a bit about the eight main building blocks of an Angular application:
+  You've learned the basics about the eight main building blocks of an Angular application:
 
-  1. [Modules](#modules)
-  1. [Components](#components)
-  1. [Templates](#templates)
-  1. [Metadata](#metadata)
-  1. [Data binding](#data-binding)
-  1. [Directives](#directives)
-  1. [Services](#services)
-  1. [Dependency injection](#dependency-injection)
+  * [Modules](#modules)
+  * [Components](#components)
+  * [Templates](#templates)
+  * [Metadata](#metadata)
+  * [Data binding](#data-binding)
+  * [Directives](#directives)
+  * [Services](#services)
+  * [Dependency injection](#dependency-injection)
 
   That's a foundation for everything else in an Angular application,
   and it's more than enough to get going.
-  But it doesn't include everything we'll need or want to know.
+  But it doesn't include everything you'll need or want to know.
 
   Here is a brief, alphabetical list of other important Angular features and services.
-  Most of them are covered in this Developers Guide (or soon will be).
+  Most of them are covered in this Developer Guide (or soon will be).
 
   > [**Animations**](animations.html): The animation library makes it easy for developers to animate component behavior
   without deep knowledge of animation techniques or CSS.
 
-  > **Change detection**: Learn how Angular decides that a component property value has changed and
-  when to update the screen.
-  Learn how it uses **zones** to intercept asynchronous activity and run its change detection strategies.
+  > **Change detection**: Read about how Angular decides that a component property value has changed,
```

Comment:
Since we don't have a link for this, we should change the text to reflect that. Maybe:

The change detection documentation will cover how...